app/controllers/katello/concerns/api/v2/hosts_controller_extensions.rb

```diff
@@ -9,6 +9,25 @@ def destroy
           sync_task(::Actions::Katello::Host::Destroy, @host)
           process_response(:object => @host)
         end
+
+        api :PUT, "/hosts/:host_id/host_collections", N_("Alter a hosts host collections")
+        param :host_id, :identifier, :required => true, :desc => N_("The id of the host to alter")
+        param :host_collection_ids, Array, :required => true, :desc => N_("List of host collection ids to update")
+        def host_collections
+          @host = ::Host::Managed.find(params[:host_id]) if params[:host_id]
```

Comment:
I don't think this find() is necessary, @host should already be defined shouldn't it?